pkg/controllers/management/drivers/cloud/cloud.go

```diff
@@ -0,0 +1,23 @@
+package clouddrivers
```

Comment:
Move this back, create a new helper that wraps the current `IsCloudDriver ` and adds a case for EKS publicaccess checking

Reply:
Moved back to `pkg/dialer/factory.go`.